### ISSUE TITLE
For OpenPower systems, renergy does not produce usage when invoked without option

### ISSUE THREAD

**Issue by whowutwut** [xcat2/xcat-core#2710]
The error message when we simply run `renergy <node>` is not valid. 

```
[root@fs1 ~]# renergy frame45cn05
Error: frame45cn05: Couldn't connect to server [50.45.5.1].
```

That seems to indicate that we cannot connect to the BMC, but in fact we are issuing the command incorrectly. .

```
[root@fs1 ~]# renergy frame45cn05 temperature
frame45cn05: CPU Temperature Instance 0           : +31 Centigrade
frame45cn05: CPU Temperature Instance 1           : +31 Centigrade
frame45cn05: Baseboard temperature Instance 0     : +25 Centigrade
```

Man page: 
```
       OpenPOWER server specific :

         renergy noderange {powerusage | temperature}
```

We should print out usage when this happens.

**Comment by whowutwut** [comment]
@immarvin  this is not OpenBMC specific, but OpenPower ....

**Comment by xuweibj** [comment]
The issue was fixed in pull request #2864 .

**Comment by xuweibj** [comment]
Hi @whowutwut , could you help to verify this issue?

**Comment by whowutwut** [comment]
I've verified on the May 3 build of xCAT 2.13.4 that issuing the above command does print the usage instead of the original message above, but it's not correct based on the help/usage information 

The current usage: 
```
    OpenPOWER server specific :
      renergy noderange [ powerusage | temperature]
```
means that `powerusage and temperature` is optional and this means that `renergy <node>` is a valid option to issue, but now it provide the usage help.. 

So we need to change either: 
1. Usage to `renergy noderange { powerusage | temperature }`   
2. Implement it to return all options if nothing is specified.  <== I would prefer this implementation

Also the usage is so verbose that it's almost pointless for a user to consume it in command line form, almost need to always look at the man page, but I'll open another issue to track that because it's a usability problem instead of a defect in the code.

**Comment by zet809** [comment]
Hi, @whowutwut , based on the priority of renergy, I think we can choose the 1st option you mentioned. 
Hi, @xuweibj , will you pls modify the usage of renergy as above? Thx a lot!

**Comment by whowutwut** [comment]
@zet809 Actually, I have been looking at this code implementation and I think number 2 makes a lot of sense, but it's more difficult to implement because we are using `powerusage` and `temperature` to identify  ipmi/BMC nodes in `Usage.pm` and then going into the ipmi.pm module....   rather i think we should be looking at the mgt=ipmi keyword to decide the plugin to invoke..  I think this is a bug in the implementation... 

So for now,...@xuweibj  please first change the Usage to show that the options are required. Then we can close this issue out.

**Comment by zet809** [comment]
Move to backlog to re-fix this issue.

**Comment by zet809** [comment]
hi, @xuweibj , pls help to update the usage of renergy. Thx!

**Comment by zet809** [comment]
The usage have been merged with PR #5332 , so I close this issue.
